src/table/components/head/HeadCellMenu.tsx

```diff
@@ -1,23 +1,35 @@
 import React, { useRef, useState, useMemo } from 'react';
 import Menu from '@mui/material/Menu';
-import More from '@qlik-trial/sprout/icons/react/More';
-import Search from '@qlik-trial/sprout/icons/react/Search';
+import More from '@qlik-trial/sprout/icons/More';
+import Search from '@qlik-trial/sprout/icons/Search';
 
 import { useContextSelector, TableContext } from '../../context';
 import { HeadCellMenuProps, MenuItemGroup } from '../../types';
-import { StyledMenuIconButton, NebulaListBox } from './styles';
-import { ListBoxWrapper, ListBoxWrapperRenderProps } from './ListBoxWrapper';
+import { StyledMenuIconButton } from './styles';
 import MenuItems from './MenuItems';
 
 export default function HeadCellMenu({ columnIndex, isDimension, tabIndex }: HeadCellMenuProps) {
-  const { translator } = useContextSelector(TableContext, (value) => value.baseProps);
+  const { translator, embed, layout } = useContextSelector(TableContext, (value) => value.baseProps);
   const [openMenuDropdown, setOpenMenuDropdown] = useState(false);
-  const [openListboxDropdown, setOpenListboxDropdown] = useState(false);
-  const anchorRef = useRef<HTMLButtonElement>(null);
+  const anchorRef = useRef<HTMLDivElement>(null);
+  const listboxRef = useRef<HTMLDivElement>(null);
+  const isMasterDimension = layout.qHyperCube.qDimensionInfo[columnIndex]?.qLibraryId;
```

Comment:
@cbt1 did we conclude that this is always the case for masterDimensions?

Reply:
also, I think this wont work for dimensions that are placed after measures. the `columnIndex` passed in here does not correspond to where the index in qDimensionInfo

Reply:
I think we should pass column instead of `columnIndex` to `HeadCellMenu`, especially since that `columnIndex` exists on column 🙃

Reply:
then we can also remove isDimension, since that is also on the column... I forgot all this when refactoring the header

Reply:
@haxxmaxx Yes I would think so.

Also I would think that storing the `isMasterDimension` on the type `Column` would make more sense

Reply:
that is a good idea!

Reply:
✅ updated